utils/test_bootstrapping/test_bootstrap_keygen.cpp

```diff
@@ -0,0 +1,175 @@
+/* Copyright (C) 2020 IBM Corp.
+ * This program is Licensed under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License. See accompanying LICENSE file.
+ */
+#include <helib/helib.h>
+#include <helib/debugging.h>
+
+#include <helib/ArgMap.h>
+
+#include <random>
+
+#include <common.h>
```

Comment:
gaps

Reply:
These are gone -- do you want me to make the same changes to the existing test? https://github.com/helibproject/HElib/blob/9f6ce7fc027d0cfc40354141752b81f864a13b61/utils/test_bootstrapping/test_bootstrap.cpp#L12